find/all/index.qmd

```diff
@@ -17,6 +17,212 @@ tidymodels_prefer()
 load("tidymodels_functions.RData")
 ```
 
+```{r}
+#| label: grey-out-s3
+#| echo: false
+# Ran in `make_function_lists.R`
+# tidymodels_functions |>
+#   filter(stringr::str_detect(functions, "\\.")) |>
+#   mutate(s3 = stringr::str_remove(functions, "\\..*")) |>
+#   filter(s3 != "") |>
+#   relocate(s3, functions) |>
+#   arrange(s3) |>
+#   View()
+
+s3generics <- c(
```

Comment:
Why is it that we inline this rather than using the results from above? Seems initially like a higher maintenance burden, though I'm totally on board if there's more to this than I'm wrapping my head around.

Reply:
Because there are some false positives for the above results. So I'm manually removing those 🧹 

It will have to be rerun every once in a while to update. But the way the original code is run, there isn't a way to make it foolproof without major overtaking

Reply:
Gotcha, that makes sense.